src/store/modules/blocks/actions.js

```diff
@@ -67,6 +103,42 @@ export default {
     return block
   },
 
+  /**
+   * Fetches the head of the blockchain
+   * @param {Object} state
+   * @param {Function} commit
+   * @param {Function} dispatch
```

Comment:
Actually, the first argument is a `store` that contains `state`, `commit`, and `dispatch` fields.